src/components/headings/H6.scss

```diff
@@ -0,0 +1,11 @@
+@import '../../styles/variables';
+
+h6.bdl-H6 {
+    @include font-family;
+
+    color: $bdl-gray-62;
+    font-size: 13px;
+    font-weight: normal;
+    letter-spacing: .3px;
+    line-height: 20px;
```

Comment:
H6 supposed to have taller line height than H5?

Reply:
according to the BDL, yes